config/application.rb

```diff
@@ -124,9 +124,8 @@ class Application < Rails::Application
       #   dev/test/prod in the config revamp.
       ActiveRecord::Base.logger.level = Logger::DEBUG
 
-      Rails::ConsoleMethods.class_eval do
-        include Vmdb::ConsoleMethods
-      end
+      Rails::ConsoleMethods.include(Vmdb::ConsoleMethods)
+      TOPLEVEL_BINDING.eval('self').extend(Vmdb::ConsoleMethods) if defined?(Pry)
```

Comment:
I don't mind hacking it in MIQ for now with a big HACK comment but yeah, it seems like something pry-rails should have if it doesn't.